app/views/questions/_eval_info.html.erb

```diff
@@ -0,0 +1,18 @@
+
+<div class="nav-info mobile-content">
+    <% if !(current_user.show_solutions) %>
+      <% if @room.state.to_s != "used" %>
+          <%if (@room.score < @room.score_limit)%>
+            <%= "Dosiahnuté skóre:"%>
```

Comment:
Ozaj a co sa tyka stringov, bolo by dobre zacat pouzivat `I18n` (Internationalization). Teraz to nevyzadujme, do hlavneho codebase sa to dostalo az v priebehu vyvoja tejto vetvy, ale nove veci je dobre robit cez to.

Cize miesto retazca napevno "Dosiahnute skore" napises napr. `t('eval.score_attained')` a do suborov `config/locales/sk.yml` a `config/locales/en.yml` pridas do kluca `eval` kluc `score_attained` a slovensky, resp. anglicky retazec).